utils/healthcheck.ts

```diff
@@ -132,12 +138,12 @@ export const checkUrlHealth = (url: string): HealthCheck<"Url", true> =>
  * @returns either true or an array of error messages
  */
 export const checkApplicationHealth = (): HealthCheck<ProblemSource, true> =>
-  taskEither
-    .of<ReadonlyArray<HealthProblem<ProblemSource>>, void>(void 0)
-    .chain(_ => checkConfigHealth())
-    .chain(config =>
+  pipe(
+    TE.of<ReadonlyArray<HealthProblem<ProblemSource>>, void>(void 0),
+    TE.chain(_ => checkConfigHealth()),
+    TE.chain(config =>
       // TODO: once we upgrade to fp-ts >= 1.19 we can use Validation to collect all errors, not just the first to happen
-      sequenceT(taskEither)<
+      AP.sequenceT(TE.taskEither)<
```

Comment:
`TE.taskEither` is deprecated. Have we an alternative? Maybe we can use `TE.ApplicativePar` instead

Reply:
Done [here](https://github.com/pagopa/io-functions-services/pull/154/commits/a889d062d9d715b4f3366ca89bc6f9814aa0abd5)